### PR TITLE
JK - Forms & Animation

### PR DESCRIPTION
**Pull request by justinkaseman**
### Description

Loads up some new dependencies:
 - Formik (form library)
- Framer Motion (animation library)
- @popmotion/popcorn (animation util library)
- @ant-design/icons (icons)

Lays out some basic components for:
- Form
- Block
- Chain
- Form

The way I see it working is by holding our Chain's state in our store (React Context). When a Block is added it will push another object on to the Chain causing a re-render. Inside this new Block will be a Form to fill out. We can have a submit button similar to Furucombo that will submit transaction of the chain to our smart contract.

### Caveats
These are basic skeletons. Feel free to change.

### CODE REVIEW

File: frontend/src/components/Block.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import React from "react";
```
Comment (fubhy) [inline frontend/src/components/Block.tsx:1]: We should use `import * as React` throughout. This is the suggested import method and will be enforced in the future: https://github.com/facebook/react/pull/18102

File: frontend/src/components/index.ts [hunk 2]
```diff
@@ -1,3 +1,7 @@
 import { ANTVersion } from "./ANTVersion";
+import { Form } from "./Form";
+import { Chain } from "./Chain";
+import { Block } from "./Block";
+import { InitialBlock } from "./InitialBlock";
 
-export { ANTVersion };
+export { ANTVersion, Form, Chain, Block, InitialBlock };
```
Comment (fubhy) [inline frontend/src/components/index.ts:7]: I don't think we should use this pattern of a single `index.ts` that re-exports all of our components. It effectively breaks code splitting.

File: frontend/src/App.tsx [hunk 3]
```diff
@@ -1,11 +1,37 @@
...
 
 const App: React.FC = () => (
-  <div>
-    <ANTVersion />
-  </div>
+  <Layout className="layout">
+    <Header>
+      <Row gutter={10}>
+        <Col>
+          <ApartmentOutlined style={{ marginTop: "10px", fontSize: "40px" }} />
```
Comment (fubhy) [inline frontend/src/App.tsx:17]: Is this the suggested way of styling Ant components or can we write actual CSS or styled-components or w/e instead? Is that possible?

Comment (justinkaseman) [reply frontend/src/App.tsx:17]: Yeah we can write actual CSS or use styled-components.

File: frontend/src/components/Block.tsx (hunk 1 above)
Comment (justinkaseman) [reply frontend/src/components/Block.tsx:1]: 👍

**Pull request merged**